ocp-build-data-validator/validator/__main__.py

```diff
@@ -10,8 +10,12 @@
 def validate(file, exclude_vpn, schema_only):
     (parsed, err) = format.validate(open(file).read())
     if err:
-        msg = '{} is not a valid YAML\nReturned error: {}'.format(file, err)
-        support.fail_validation(msg, parsed)
+        print(f'{file} is not a YAML file')
```

Comment:
This will make the validator completely ignore syntax errors in yaml files. We need to either exclude non-yaml files from the invocation, or determine if a file is yaml or not in a better way.

Reply:
updated, used an other way to check